pom.xml

```diff
@@ -362,7 +363,7 @@
       <dependency>
         <groupId>org.jctools</groupId>
         <artifactId>jctools-core</artifactId>
-        <version>1.2</version>
+        <version>1.2.1</version>
```

Comment:
Yes

> Am 25.06.2016 um 18:57 schrieb Scott Mitchell <notifications@github.com>:
> 
> In pom.xml:
> 
> > @@ -362,7 +363,7 @@
> >        <dependency>
> >          <groupId>org.jctools</groupId>
> >          <artifactId>jctools-core</artifactId>
> > -        <version>1.2</version>
> > +        <version>1.2.1</version>
> With this update I believe we can now update MAX_ALLOWED_MPSC_CAPACITY to be 1 << 30? @normanmaurer - please confirm.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>